File: non repeating element/nonrepeat.py
#Given an array of N integers, and an integer K, the task is to find the number of pairs of integers in the array whose sum is equal to K.

def pair(arr,k):
    count = 0
    for i in range(len(arr)):
        for j in range(i+1,len(arr)):
            if arr[i]+arr[j] == k:
                count += 1
    return count
print(pair([1,5,7,1],6))

File: non repeating element/nonrepeat.js
//Find first non-repeating element in a given Array of integers
function firstnonrepeat(arr){
    let obj={};
    for (let i=0;i<arr.length;i++){
        if(obj[arr[i]]){
            obj[arr[i]]+=1;
        }
        else{
            obj[arr[i]]=1;
        }
    }
    for (let i=0;i<arr.length;i++){
        if (obj[arr[i]]==1){
            return arr[i];
        }
    }
    return "No non-repeating element";
    
}
console.log(firstnonrepeat([1,2,3,4,5,6,7,8,9,1]));